### PR TITLE
Fix a typo and a broken link

### PR DESCRIPTION
**Pull request by taaaf11**
Fix a typo and a broken link.

### CODE REVIEW

**Comment by taaaf11** [comment]
@CraigglesO Kindly review my pull request

File: package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
     "name": "everforest-moist",
     "displayName": "everforest-moist",
-    "description": "This theme is a blended version of the original everfoest themes.",
+    "description": "This theme is a blended version of the original everforest themes.",
     "version": "0.0.2",
     "LICENSE": "ISC",
     "publisher": "MrMartian",
     "repository": {
       "type": "git",
-      "url": "https://github.com/CraigglesO/everforest-moist-vscode"
```
Comment (CraigglesO) [inline package.json:10]: Thanks for the effort. however, this doesn't link to anything anymore. https://github.com/CraigglesO/everforest-moist-vscode doesn't exist.

Comment (taaaf11) [reply package.json:10]: @CraigglesO That's why I have changed the code to point to correct link.

Comment (CraigglesO) [reply package.json:10]: Right, when I copy paste the URL you have changed to: https://github.com/CraigglesO/everforest-moist-vscode
I get a 404. The original which is https://github.com/CraigglesO/everforest-moist works however. Can you change it back? Click the links to see what I mean

Comment (taaaf11) [reply package.json:10]: @CraigglesO I haven't changed it to `everforest-moist-vscode`, rather I changed it to `everforest-moist`. That `-` sign appearing at the start of line means that line was "deleted", and immediately after that, the `+` sign means that the change I made. Overall, it means that the line was changed to contain `everforest-moist` from `everforest-moist-vscode`.

Comment (CraigglesO) [reply package.json:10]: Wow, Sorry I got that mixed up -_- thank you so much for the clarification.

Comment (taaaf11) [reply package.json:10]: No worries 😊

**Pull request merged**